jvm-js/main/scala/org/scalacheck/time/JavaTimeArbitrary.scala

```diff
@@ -25,14 +25,11 @@ private[scalacheck] trait JavaTimeArbitrary {
   // Duration.MAX/Duration.MIN values, but in practice they are finite,
   // restricted by their underlying representation a long and an int.
 
-  private final lazy val minJavaDuration: Duration =
-    Duration.ofSeconds(Long.MinValue)
-
-  private final lazy val maxJavaDuration: Duration =
-    Duration.ofSeconds(Long.MaxValue, 999999999L)
-
-  implicit final lazy val arbJavaDuration: Arbitrary[Duration] =
+  implicit final lazy val arbJavaDuration: Arbitrary[Duration] = {
+    val minJavaDuration = Duration.ofSeconds(Long.MinValue)
+    val maxJavaDuration = Duration.ofSeconds(Long.MaxValue, 999999999L)
     Arbitrary(Gen.choose(minJavaDuration, maxJavaDuration))
+  }
```

Comment:
The Scala.js linker/optimizer was unable to elide the `Duration` reference for some reason. This fixed it.